addons/purchase/models/purchase.py

```diff
@@ -402,35 +399,126 @@ def _add_supplier_to_product(self):
                 except AccessError:  # no write access rights -> just ignore
                     break
 
-    def action_view_invoice(self):
-        '''
-        This function returns an action that display existing vendor bills of given purchase order ids.
-        When only one found, show the vendor bill immediately.
+    def action_create_invoice(self):
+        """Create the invoice associated to the PO.
+        :param grouped: if True, invoices are grouped by PO id. If False, invoices are grouped by
```

Comment:
now you removed the param you can remove the doc about it :eyeglasses:

Reply:
hahaha, right